Automattic/sensei
# Fix students context menu

### Changes proposed in this Pull Request

* Update `@wp-components` to version `wp-6.1`

With the old component, there is a problem with the students context menu not showing properly.  Updating to the latest component fixes that issue.

### Testing instructions

* Go to Sensei -> Students
* Click the `...` menu
* Make sure it opens up

Example:  
![student-context-menu](https://user-images.githubusercontent.com/3220162/202825245-cf52fa13-d90f-4195-b34c-672580357d88.gif)


Also, I wasn't sure if I needed to push `package-lock.json`?   It wasn't in `.gitignore` so.... let me know!
